vpdq/CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+If you would like to contribute to vPDQ, you can do the following:
+
+[Fork][fork] the repo and then [clone][clone] it locally:
+
+[fork]: https://help.github.com/articles/fork-a-repo/
+[clone]: https://help.github.com/articles/which-remote-url-should-i-use/
+
+    git clone git@github.com:your-username/ThreatExchange
+
+## Branch locally and develop!
+Make a branch in your cloned fork. We suggest naming the branch by feature name
+or “issue_XX” where XX is the issue number the branch is associated with. Make
+your changes in your branch and test thoroughly. If this is a large feature you
+can push your branch to your fork often. This allows you to request feedback for
+how things are progressing instead of dumping a large code change all at once.
+
+When making commits to your branch, make sure you write [well-formed][wf] commit
+messages and update documentation accordingly (see the next section).
+
+[wf]: https://github.com/erlang/otp/wiki/Writing-good-commit-messages
+
+### Code Style
+vPDQ uses [black](https://pypi.org/project/black/) for consistent formatting across
+the projects Python source files and clang-format for formating CPP source files. 
+After installing black and clang-format locally, you can automatically format all the vpdq files by running the following command from the repository root.
+
+```shell
+black ./vpdq/
+find vpdq/ -iname '*.h' -o -iname '*.cpp' | xargs clang-format -i
+```
+
+### Installing Locally
+vPDQ has two parts of codes: CPP implementation and python-binding library.
+To use vPDQ, ffmpeg is required(https://www.ffmpeg.org/download.html)
+
+To cmake CPP binary files:
+```
+cd vpdq/cpp/
+mkdir -p build
+cd build
+cmake ..
+make
+```
+To locally install python-binding library:
```

Comment:
aside: have you determined a python version requirements?

Reply:
I think both 3.8 and 3.9 work, but I haven't tested for other versions.

Reply:
Do you think it is something worth specifying here or in the READ.md ?